script/publish_views

```diff
@@ -4,4 +4,8 @@
 
 cd "$(dirname "$0")/.."
 
-exec python3 -m bigquery_etl.view.publish_views "$@"
+./bqetl bootstrap
```

Comment:
why does this need to bootstrap every time it's run?

Reply:
Running `./bqetl bootstrap` will install requirements and setup `venv`. [Jenkins](https://github.com/mozilla-services/cloudops-infra/blob/623a8cc546d0084d837dbf4889d1239d562d4922/projects/data-pioneer/Jenkinsfile.bigquery#L243) will run this script and without running bootstrap `./bqetl view publish` won't work in that case.